packages/o365_metrics/data_stream/active_users/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,105 @@
+---
+description: Pipeline for renaming object
+processors:
+  - set:
+      field: ecs.version
+      value: "8.16.0"
+  - json:
+      field: active_users
+      target_field: o365.metrics.active.count.users   
+  - script:
+      lang: painless
+      description: Replace spaces and dashes in field names under o365.metrics.active.count.users.
+      tag: painless_purge_spaces_and_dashes
+      if: ctx.o365.metrics?.active?.count?.users instanceof Map
+      source: |
+        String underscore(String s) {
+          return /[ -]/.matcher(s).replaceAll('_').toLowerCase();
```

We need to update this, as the JSON in Elasticsearch contains a BOM character, which needs to be handled.
```suggestion
          String result = /[ -]/.matcher(s).replaceAll('_').toLowerCase();
          return /[\ufeff]/.matcher(result).replaceAll('')
```